game.py

```diff
@@ -12,7 +12,17 @@ class Random(Bot):
     """
     Cory's bot :)
     """
-    pass
+
+    def move(self, state):
+        valid_pile_indexes = []
+        for i, pile in enumerate(state):
+            if pile >=1:
+                valid_pile_indexes.append(i)
+        if not valid_pile_indexes:
+            raise NimException(f'No valid moves on state: {state}')
```

Comment:
This is a great exception message. You should update garbage bot to use it too... https://media1.tenor.com/images/c9fd3f5ad87f9c9ae12a8e70aa0d14a5/tenor.gif?itemid=9639526